README.md

```diff
@@ -40,16 +40,16 @@ Please follow the [ESPHome documentation](https://esphome.io/guides/getting_star
 
 ### Pin-out
 
-| RJ45 pin | Name      | ESP32        |
-| -------- | --------- | ------------ |
-| 8        | +5V (VDD) | VIN          |
-| 7        | GND       | GND          |
-| 6        | TX        | TX2 (GPIO17) |
-| 5        | RX        | RX2 (GPIO16) |
-| 4        | PIN 20    | D23 (GPIO23) |
-| 3        | (unknown) |              |
-| 2        | SWIM      |              |
-| 1        | RES       |              |
+| RJ45 pin | Name      | ESP32        | ESP32-C3     |
```

Comment:
These ESP32 pins match `packages/office-desk-esp32.yaml`. If you want to add alternative pins for ESP32-C3, then it's probably worth adding an ESP32-C3 version of that file too.

Reply:
Added and linked to the files from the tables https://github.com/cherniavskii/LoctekMotion_IoT/blob/esp32c3/README.md